x-pack/plugins/reporting/server/lib/screenshots/constants.ts

```diff
@@ -4,8 +4,10 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
+// import { APP_WRAPPER_CLASS } from '../../../../../../src/core/public';
+// export const DEFAULT_PAGELOAD_SELECTOR = `.${APP_WRAPPER_CLASS}`; // TODO: Doesn't like this, server crashes
 
-export const DEFAULT_PAGELOAD_SELECTOR = '.application';
+export const DEFAULT_PAGELOAD_SELECTOR = '.kbnAppWrapper'; // TODO: Can we use the var? ^^
```

Comment:
@tsullivan This PR makes a reusable selector name for our outer wrapper via a global var called `APP_WRAPPER_CLASS`. I tried the above commented code, but it completely broke the dev server.

Reply:
The import should be from a `server` location. If we change `src/core/server/index.ts` to export this variable, we can import it from there.

Reply:
Thank you! Done and works!